### ISSUE TITLE
SQS Source: time.Time: not in RFC3339 format

### ISSUE THREAD

**Issue by AceHack** [knative/eventing-contrib#993]
**Describe the bug**
When trying to send to in-memory channel or another knative service from SQS Source I receive the following error.

{"level":"error","ts":1583510808.2513373,"logger":"fallback","caller":"http/transport.go:572","msg":"failed to convert http message to event","error":"cannot convert \"52149-06-15T07:22:46Z\" to time.Time: not in RFC3339 format","stacktrace":"knative.dev/eventing-contrib/vendor/github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http.(*Transport).ServeHTTP\n\t/home/prow/go/src/knative.dev/eventing-contrib/vendor/github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http/transport.go:572\nnet/http.(*ServeMux).ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2387\nnet/http.serverHandler.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2802\nnet/http.(*conn).serve\n\t/usr/local/go/src/net/http/server.go:1890"}

 **Expected behavior**
I expect the cloud event to be successfully transmitted to the sink

 **To Reproduce**
Create Knative service
```
apiVersion: serving.knative.dev/v1alpha1
kind: Service
metadata:
  name: awssqs-event-display
  namespace: content-platform-sqs-source
spec:
  runLatest:
    configuration:
      revisionTemplate:
        spec:
          container:
            image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display@sha256:b2c341b2f36c46ed20041364e19cdaa19c7002dad97646f69717257b5afa4512
```

Create Knative SQS event source
```
apiVersion: sources.eventing.knative.dev/v1alpha1
kind: AwsSqsSource
metadata:
  name: awssqs-sample-source
  namespace: content-platform-sqs-source
spec:
  awsCredsSecret:
    name: aws-credentials
    key: credentials
  queueUrl: "https://sqs.us-east-1.amazonaws.com/xxxx/sqs-test"
  sink:
    apiVersion: serving.knative.dev/v1alpha1
    kind: Service
    name: awssqs-event-display
```

push sqs event
Either command line or UI

Error now happens

 **Knative release version**
Serving: v0.11.1
Eventing: v0.11.0
SQS Source v0.11.2

 **Additional context**
We are in EKS so we can only use Kubernetes 1.14 for now, so I would really like to see a fix in 0.11.x since we can't upgrade to 0.12.x yet because of EKS.

This is a blocking issue for us currently.

Thanks.

**Comment by AceHack** [comment]
I've created a PR that should fix the issue, could someone approve it?  Thanks so much for your help.

**Comment by AceHack** [comment]
Closed with #996
